ios/browser/browser_state/BUILD.gn

```diff
@@ -7,14 +7,19 @@ import("//build/config/ios/rules.gni")
 import("//ios/build/config.gni")
 
 source_set("browser_state") {
+  configs += [ "//build/config/compiler:enable_arc" ]
   sources = [
-    "brave_browser_state_keyed_service_factories.cc",
     "brave_browser_state_keyed_service_factories.h",
+    "brave_browser_state_keyed_service_factories.mm",
   ]
   deps = [
     "//base",
+    "//brave/components/brave_component_updater/browser",
+    "//brave/components/https_upgrade_exceptions/browser",
```

Comment:
why is this added?

Reply:
actually none of these should be added except for `brave/ios/browser/url_sanitizer`

Reply:
just something I forgot to remove after adding that brave browser state manager. Sorry about that.

Reply:
Removed